elements/elements.md

```diff
@@ -3,7 +3,6 @@ helium
 carbon is a metal? 
 thorium 
 protactinium
```

Comment:
I like that you added a folder for the elements file. But Uranium is still an element. Could you please add it back in to the file?

Reply:
got it but I do think it should be at the bottom cus its so heavy